paasta_tools/kubernetes_tools.py

```diff
@@ -775,7 +780,7 @@ def get_kubernetes_services_running_here() -> Sequence[KubeService]:
     services = []
     pods = requests.get('http://127.0.0.1:10255/pods').json()
     for pod in pods['items']:
-        if pod['status']['phase'] != 'Running' or pod['metadata']['namespace'] != 'paasta':
+        if pod['status']['phase'] != 'Running' or 'smartstack_registrations' not in pod['metadata']['annotations']:
```

Comment:
`annotations` is optional in `pod['metadata']`, the code probably should be more defensive here trying not to trigger `KeyError`.